app/controllers/observations_controller.rb

```diff
@@ -80,4 +80,11 @@ def observation_params
           :id, :station_id, :direction, :speed, :min_wind_speed, :max_wind_speed
       )
     end
+
+    def expiry_time(station)
+      t = station.next_observation_expected_in
+      # checks if station is overdue for reporting in which case it sets the
+      # expiry to 1 minute to encourage the client to check again.
```

Comment:
Please clarify which client is meant since we both have both web browsers and the arduino wind stations that acts as clients to the server. I understand that it is the web browser that is meant but it would be good to clarify.